antlr/src/main/antlr4/org/apache/iotdb/db/qp/strategy/SqlBase.g4

```diff
@@ -40,8 +40,9 @@ statement
     | DELETE STORAGE GROUP prefixPath (COMMA prefixPath)* #deleteStorageGroup
     | SHOW METADATA #showMetadata // not support yet
     | DESCRIBE prefixPath #describePath // not support yet
-    | CREATE INDEX ON fullPath USING function=ID indexWithClause? whereClause? #createIndex //not support yet
-    | DROP INDEX function=ID ON fullPath #dropIndex //not support yet
+    | CREATE INDEX ON prefixPath whereClause? indexWithClause #createIndex //not support yet
```

Comment:
If you support this SQL, you should delete `//not support yet`

Reply:
> If you support this SQL, you should delete `//not support yet`

In the current version, the SQL about the index query will throw an "UnSupportedOperation" exception. Shall we remove "not support yet" now, or defer to delete it until the function is fully supported?

Reply:
ok, you can delete it until the function is fully supported.

Reply:
> ok, you can delete it until the function is fully supported.

Sure